ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Node.hs

```diff
@@ -315,58 +315,22 @@ protocolInfoShelleyBased ProtocolParamsShelleyBased {
     initLedgerState :: LedgerState (ShelleyBlock era)
     initLedgerState = ShelleyLedgerState {
         shelleyLedgerTip        = Origin
-      , shelleyLedgerState      = SL.chainNes initShelleyState
+      , shelleyLedgerState      =
+          registerGenesisStaking (SL.sgStaking genesis) $
+            SL.initialState genesis ()
       , shelleyLedgerTransition = ShelleyTransitionInfo {shelleyAfterVoting = 0}
       }
 
     initChainDepState :: TPraosState (EraCrypto era)
     initChainDepState = TPraosState Origin $
-      SL.ChainDepState {
-          SL.csProtocol = SL.PrtclState
-            (SL.chainOCertIssue     initShelleyState)
-            (SL.chainEvolvingNonce  initShelleyState)
-            (SL.chainCandidateNonce initShelleyState)
-        , SL.csTickn = SL.TicknState
-            (SL.chainEpochNonce     initShelleyState)
-            (SL.chainPrevEpochNonce initShelleyState)
-        , SL.csLabNonce =
-            (SL.chainPrevEpochNonce initShelleyState)
-        }
-
-    initialEpochNo :: EpochNo
-    initialEpochNo = 0
-
-    -- We can just use 'SL.genesisUTxO' to compute the initial UTxO from the
-    -- initial funds. No need to combine the initial funds with an existing
-    -- UTxO, so no need for 'registerInitialFunds'.
-    initialUtxo :: SL.UTxO era
-    initialUtxo = SL.genesisUTxO genesis
-
-    initShelleyState :: SL.ChainState era
-    initShelleyState =
-        overNewEpochState
-          (registerGenesisStaking (SL.sgStaking genesis)) $
```

Comment:
After some debugging today, we realized the hydra build is failing because this line was lost in the diff.

Reply:
Now added back at https://github.com/input-output-hk/ouroboros-network/pull/2946/files#diff-ad1bcf5864925856d8007dccad41cda0a70518e25040c4b8a70bb760909ef644R319